appveyor.yml

```diff
@@ -157,6 +160,25 @@ test_script:
      $wc = New-Object 'System.Net.WebClient'
      $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", $unitTestsXml)
      if($errorCode -ne 0) { $host.SetShouldExit($errorCode) }
+ - ps: |
+    if($env:APPVEYOR_PULL_REQUEST_NUMBER) {
+      $lintOutput = (Resolve-Path .\testOutput\lint\)
+      $lintSource = (Resolve-Path .\tests\lint\)
+      git fetch -q origin $env:APPVEYOR_REPO_BRANCH
+      $prDiff = "$lintOutput\prDiff.patch"
+      git diff -u FETCH_HEAD...HEAD > $prDiff
+      $flake8Config = "$lintSource\flake8.ini"
+      $flake8Output = "$lintOutput\PR-Flake8.txt"
+      type "$prDiff" | py -m flake8 --exit-zero --disable-noqa --diff --output-file="$flake8Output" --tee --config="$flake8Config"

```

Comment:
Is your usage of --exit-zero intentional?

Reply:
Yes, for now, I'm testing it out. It would be nice if lint errors did not stop the build from completing, so we get access to build artifacts etc. But we still need to be notified of the failures.

Reply:
First I want to confirm that this is the thing stopping the build with a mysterious "Command exited with code 1" after the system tests are done.